traffic_portal/package-lock.json

```diff
@@ -5,6 +5,7 @@
   "requires": true,
   "packages": {
     "": {
+      "name": "traffic_portal",
```

Comment:
I don't understand why this wasn't added a long time ago, the name field in `package.json` has been set for like 4 years.